bitnami/elasticsearch/values.yaml

```
@@ -1082,6 +1082,13 @@ metrics:
     # pullSecrets:
     #   - myRegistryKeySecretName
 
+  ## Extra arguments for the exporter
+  ## ref: https://github.com/justwatchcom/elasticsearch_exporter
+  ##
+  # extraArgs:
+  #   - --es.snapshots
+  #   - --es.indices
```

Use the example in the comments, and set a empty array by default value:
```suggestion
  ## Extra arguments for the exporter
  ## ref: https://github.com/justwatchcom/elasticsearch_exporter
  ## e.g
  ## extraArgs:
  ##   - --es.snapshots
  ##   - --es.indices
  ##
  extraArgs: []
```